### PR TITLE
coreftp: Add version 2.2

### PR DESCRIPTION
**Pull request by wenmin92**
[coreftp](http://www.coreftp.com/) is a free FTP client.

### CODE REVIEW

File: bucket/coreftp.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            "corecmd.exe"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "coreftpu.exe",
+            "Core FTP"
+        ]
+    ],
+    "post_install": "Remove-Item \"$dir\\uninstall.exe\" -Force",
```
Comment (Ash258) [inline bucket/coreftp.json:32]:
```suggestion
```

File: bucket/coreftp.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            "hash": "20db16ef37a5dc26016802172518da1e50a8e7a90cf75828fccf9474df2fffde"
+        }
+    },
+    "bin": [
+        [
+            "coreftpu.exe",
+            "coreftp.exe",
+            "corecmd.exe"
+        ]
+    ],
```
Comment (Ash258) [inline bucket/coreftp.json:25]: Any of this executable is working from command line. Post screenshot of any command execution with parameters and result.

```suggestion
    "pre_install": "Remove-Item \"$dir\\uninst*\" -Force",
    "bin": [
        "coreftpu.exe",
        "coreftp.exe",
        "corecmd.exe"
    ],
```

File: bucket/coreftp.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{
+    "version": "2.2",
+    "homepage": "http://www.coreftp.com",
+    "description": "A FTP client",
+    "license": {
+        "identifier": "freeware",
```
Comment (Ash258) [inline bucket/coreftp.json:6]:
```suggestion
        "identifier": "Freeware",
```

File: bucket/coreftp.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
+{
+    "version": "2.2",
+    "homepage": "http://www.coreftp.com",
+    "description": "A FTP client",
```
Comment (Ash258) [inline bucket/coreftp.json:4]:
```suggestion
    "description": "A FTP client",
    "homepage": "http://www.coreftp.com",
```

File: bucket/coreftp.json (hunk 2 above)
Comment (wenmin92) [reply bucket/coreftp.json]: I have deleted `coreftp.exe` and `corecmd.exe`, and renamed `coreftpu.exe` to `coreftp.exe`.

**Comment by wenmin92** [comment]
According to the origin installer commands, I have changed the installer script.
<details>
  <summary>installer commands</summary>

```
Output folder: C:\Program_Files\CoreFTP
Extract: ccerts.dll... 100%
Extract: libeay32.dll... 100%
Extract: ssleay32.dll... 100%
Extract: ssh.dll... 100%
Extract: pftpns.dll... 100%
Extract: corecmd.exe... 100%
Extract: ascii.txt... 100%
Extract: coreftp.chm... 100%
Extract: coreftp.exe
Created uninstaller: C:\Program_Files\CoreFTP\uninstall.exe
Create folder: C:\Users\wenmin92\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Core FTP (x64)
Create shortcut: C:\Users\wenmin92\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Core FTP (x64)\Uninstall.lnk
Create shortcut: C:\Users\wenmin92\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Core FTP (x64)\Core FTP LE.lnk
Create shortcut: C:\Users\wenmin92\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Core FTP (x64)\Core FTP Help.lnk
Delete file: C:\Program_Files\CoreFTP\coreftp.exe
Extract: coreftpu.exe... 100%
Rename: C:\Program_Files\CoreFTP\coreftpu.exe->C:\Program_Files\CoreFTP\coreftp.exe
Extract: SFTPDll64.dll... 100%
Rename: C:\Program_Files\CoreFTP\sftpdll64.dll->C:\Program_Files\CoreFTP\sftpdll.dll
Completed
```
</details>

File: bucket/coreftp.json [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        "32bit": {
+            "url": "http://www.coreftp.com/download/coreftplite.exe#/dl.7z",
+            "hash": "20db16ef37a5dc26016802172518da1e50a8e7a90cf75828fccf9474df2fffde"
+        }
+    },
+    "installer": {
+        "script": [
+            "Remove-Item \"$dir\\uninst*\", \"$dir\\coreftp.exe\" -Force",
+            "Rename-Item \"$dir\\coreftpu.exe\" \"coreftp.exe\" -Force",
+            "Rename-Item \"$dir\\sftpdll64.dll\" \"sftpdll.dll\" -Force"
```
[the remark this answers is not in the document]
Comment (wenmin92) [reply bucket/coreftp.json]: Yes, you are right. I have move the third line to architecture scope. But I don't have a 32bit system, I can't verify this. Maybe someone install this app and if not work, we then fix.

**Comment by mark05e** [comment]
This is unrelated to Issue https://github.com/lukesampson/scoop-extras/issues/4375

**Pull request merged**